app/scripts/controllers/gene.js

```diff
@@ -993,13 +993,13 @@ angular.module('oncokbApp')
                     subtype: null,
                     articles: [],
                     treatments: null,
-                    propagation: null
+                    solidPropagationLevel: null,
```

Comment:
We should make it a standard how the naming should be used. Let's stick with `solidPropagation`. Could you change `propagationLiquid_review ` to `liquidPropagation_review `?

Reply:
I guess it's because for the simplicity to use `propagation` in the treatment model as solidPropagation? If that's the case, we can stick with propagationLiquid as well. Just want to make sure only one format should be used.

Reply:
@zhx828 I think we'd better use the identical name with Evidence model. That's prepared to post data to `/evidences/update` api.

Reply:
@victoria34 sure, but should we update the keyword `propagationLiquid_review ` to `liquidPropagation_review ` and even update `propagation` to `solidPropagation`?